mongo/description/shared_spec_test.go

```diff
@@ -207,7 +207,7 @@ func selectServers(t *testing.T, test *testCase) error {
 
 		// Max staleness can't be sent to servers older than 3.4.
```

Comment:
Update the comment?

Reply:
I think removing the comment works. I'm not sure this block is even relevant today. I can't think of a case where the server's wire version should be nil.